### DISCUSSION TITLE
I'm using the Github Project

### DISCUSSION THREAD

**Discussion by ridsync** [community/community#16237] [Other Feature Feedback, Questions, & Ideas]
Hello
I'm using the Github Project(New).
It's still beta, but I'm going to use it because I think it'll be enough to manage the project.
I understand that it is still under development, and I have some suggestions regarding it.

- Please allow each item to be sorted by issue, PR generation time, or modification time.
- Also, it would be good to see the related PR or issue directly on the same screen. Currently, it seems to be only open with a browser tab.

Have a nice day
Thank you.

**Comment by queenofcorgis** [comment]
Thanks for your feedback!
